libs/components/phone-field/testing/src/phonefield/phone-field-fixture.spec.ts

```diff
@@ -170,13 +171,26 @@ describe('PhoneField fixture', () => {
       'selectedCountryChange'
     );
 
-    // change the country
-    await phonefieldFixture.selectCountry(COUNTRY_AU.name);
+    if (COUNTRY_AU.name) {
+      // change the country
+      await phonefieldFixture.selectCountry(COUNTRY_AU.name);
+    }
+
+    const countryName: string | null =
+      await phonefieldFixture.getSelectedCountryName();
+
+    const countryIos2: string | null =
+      await phonefieldFixture.getSelectedCountryIso2();
 
     fixture.detectChanges();
     await fixture.whenStable();
 
-    expect(testComponent.selectedCountry.name).toBe(COUNTRY_AU.name);
+    if (COUNTRY_AU?.name && countryName) {
+      expect(countryName).toBe(COUNTRY_AU.name);
+    }
+    if (COUNTRY_AU?.iso2 && countryIos2) {
+      expect(countryIos2).toBe(COUNTRY_AU.iso2);
+    }
```

Comment:
these checks aren't necessary, but it doesn't hurt to add them so that code coverage is 100%